src/backend/TrafficCourts/Workflow.Service/Consumers/DisputeApprovedConsumer.cs

```diff
@@ -22,7 +22,7 @@ public async Task Consume(ConsumeContext<DisputeApproved> context)
         {
             try
             {
-                if (context.RequestId != null)
+                if (context.MessageId != null)
```

Comment:
That's true. We actually only need it for the log message after giving it a second thought since the consumer will only be triggered if there is a message of the type. Will change it to what you suggested.